_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
Yeah, we already use `fluid: true`, which defaults to 16x9? So it sounds like the issue is that it's sized by javascript too late. Does this issue happen on the site today, before moving the player init into home.js instead of after the video tag?